react/week2/todo/src/App.js

```diff
@@ -1,24 +1,24 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from "react";
 import "./App.css";
 import data from "./data/data.json";
 import Header from "./components/Header";
 import Timer from "./components/Timer";
 import TodoList from "./components/TodoList";
 
 function App() {
-  // hooks
-  const [state, setState] = useState(null);
-  if (state === null) setTimeout(() => setState(data), 3000);
-
+  const [time, setTime] = useState(null);
```

Comment:
In reality you are not setting the time here but you eventually store the todo items in this state. So this could be `todoItems` or just `items`.